src/controls/polyControl.ts

```diff
@@ -90,19 +81,15 @@ export const factoryPolyActionHandler = (
       anchorPointInParentPlane = anchorPoint
         .subtract(poly.pathOffset)
         .transform(poly.calcOwnMatrix()),
-      actionPerformed = fn(eventData, { ...transform, pointIndex }, x, y),
-      adjustFlip = new Point(poly.flipX ? -1 : 1, poly.flipY ? -1 : 1);
+      actionPerformed = fn(eventData, { ...transform, pointIndex }, x, y);
 
-    const newPositionNormalized = anchorPoint
+    const newAnchorPointInParentPlane = anchorPoint
       .subtract(poly.pathOffset)
-      .divide(poly._getNonTransformedDimensions())
-      .multiply(adjustFlip);
+      .transform(poly.calcOwnMatrix());
 
-    poly.setPositionByOrigin(
-      anchorPointInParentPlane,
-      newPositionNormalized.x + 0.5,
-      newPositionNormalized.y + 0.5
-    );
+    const diff = newAnchorPointInParentPlane.subtract(anchorPointInParentPlane);
+    poly.left -= diff.x;
+    poly.top -= diff.y;
```

Comment:
@ShaMan123 this approach with diffing the position of the anchor point in a reference plance, before and after the action works. Now also skew works fine.

Reply:
nice!